### ISSUE TITLE
"onBackgroundClick" and "onEscapeKeydown" warnings

### ISSUE THREAD

**Issue by MadalenaGoncalves** [AlexanderRichey/styled-react-modal#4]
After updating to "1.0.0", the following warnings started showing up:

```
Warning: Unknown event handler property `onEscapeKeydown`. It will be ignored.
    in div (created by Context.Consumer)
    in StyledComponent (created by styled.div)
    in div (created by Context.Consumer)
```
```
Warning: Unknown event handler property `onBackgroundClick`. It will be ignored.
    in div (created by Context.Consumer)
    in StyledComponent (created by styled.div)
    in div (created by Context.Consumer)
```

I think this could be fixed by changing the extraction of props in the Modal render method. Instead of:
`    const { WrapperComponent, children, ...rest } = this.props`
it should be:
`    const { WrapperComponent, children, onEscapeKeydown, onBackgroundClick, ...rest } = this.props`

Makes sense?

**Comment by MadalenaGoncalves** [comment]
This might actually be the issue: https://github.com/styled-components/styled-components/issues/1969#issuecomment-426943214

**Comment by AlexanderRichey** [comment]
I'm not able to reproduce this issue, though I think your explanation makes sense. Can you link to a codesandbox where I can see the issue? Or maybe you could paste in some of your code so that I can get a better idea of what's going on?

**Comment by MadalenaGoncalves** [comment]
Thanks for your quick response. Here's a codesandbox: https://codesandbox.io/s/7k483z41m0
But I believe the issue is actually with styled-components v4 (see issue above), which I am currently using (also in the sandbox).

**Comment by AlexanderRichey** [comment]
Fixed by @erickjth in #5
